docs/installation.md

```diff
@@ -50,17 +50,30 @@ list of instruments:
         "platformType": "ILLUMINA",
         "name": "default",
         "timeZone": "America/Toronto",
-        "parameters": {}
+        "parameters": {},
+        "ignoreSubdirectories": [
+            "/run1"
```

Comment:
is the preceding `/` a requirement or will `run1` sans `/` work?

Reply:
Then probably write the examples without `/`. To a new user, these might look like absolute paths despite the 'subdirectories' name.